README.md

```diff
@@ -31,28 +33,63 @@ go test trident.li/pitchfork/lib -v
 go test trident.li/pitchfork/ui -v
 ```
 
-One can also run tests individually by specifying a filter, eg:
+Tests may also be individually executed by specifying a filter, eg:
 ```
 go test trident.li/pitchfork/lib -v -run IPtrk
 ```
 which would run only the iptrk related tests.
 
-The argument to ```-run``` is a regexp, ```AB[CD]``` would for instance match functions named
-```Test_ABC``` + ```Test_ABD```.
+The argument to ```-run``` is a regexp, ```AB[CD]``` would for instance
+match functions named ```Test_ABC``` + ```Test_ABD```.
 
-See also the top of the *._test.go files for the simple cut&paste variants.
+See also the top of the *._test.go files for simple cut&paste variants.
 
 ### CLI Tests
 
 Either make mini tests for the exact functions.
-Or Call pf.Cmd() passing the various that need to be done.
+Or Call pf.Cmd() passing the various operations that need to be done.
 Error or return body can then be checked.
 
 ### UI Tests
 
-Pitchfork's URLTest module (```ui/urltest/```) contains the URL_Test() function that
-accepts a URLTest structure that allows passing in various variables that act as the
-request or as the response checks. One can do positive and negative checks with it.
+Pitchfork's URLTest module (```ui/urltest/```) contains the URL_Test()
+function that accepts a URLTest structure which allows passing in 
+variables that act as the request or as the response checks, positive
+and negative checks may be performed.
 
-Passing a set Username in the test causes a cookie to be created for that user and thus
-it automatically looks like one is logged in as that user.
+Passing a set Username in the test causes a cookie to be created for that
+user and it automatically looks as if that user has logged into the system.
+
+### Prerequisite GoLang Libraries
+
+There are several prerequisite libraries which must be installed prior to
+testing pitchfork, or building it. The list is:
```

Comment:
Golang has this very useful 'go get' command that resolves these.
Noting that just getting the list you specify does not resolve the dependencies of these items or the dependencies of those items.

You might to check doc/Trident.md which has a variety of these details.